pkg/federatedtypes/configmap.go

```diff
@@ -103,7 +103,7 @@ func (a *FederatedConfigMapAdapter) ObjectForCluster(template, override pkgrunti
 	return configMap
 }
 
-func (a *FederatedConfigMapAdapter)ObjectForUpdateOp(desiredObj, clusterObj pkgruntime.Object) pkgruntime.Object {
```

Comment:
(No action required) Again, in future please ensure that functional changes are separated from non-functional.

Reply:
This is the `gofmt` automatic formatting at work.